packages/react-storage/src/components/StorageBrowser/actions/handlers/utils.ts

```diff
@@ -50,3 +52,43 @@ export const constructBucket = ({
   bucketName: string;
   region: string;
 } => ({ bucketName, region });
+
+export const parseLocationAccess = (location: LocationAccess): LocationData => {
+  const { permission, scope, type } = location;
+  if (!scope.startsWith('s3://')) {
+    throw new Error(`Invalid scope: ${scope}`);
+  }
+
+  const id = crypto.randomUUID();
+
+  // remove default path
+  const sanitizedScope = scope.slice(5);
```

Comment:
Nit: Is there a better word than sanitized? Sanitizing usually implies encoding special characters or trimming whitespace etc? `trimmedScope` maybe?

Reply:
Don't really feel strongly either way, can be done in a follow up